### PR TITLE
feat: new library GCP Icons

### PR DESCRIPTION
**Pull request by excalibot**
Here are some of the main GCP products !

Submitted by @clementbosc

Items: GCP, Data Studio, Cloud Spanner, Dataflow, BigQuery, Data Transfer Service, Pub/Sub, Cloud Storage, Cloud Workflow, Cloud Scheduler, Cloud Build, Cloud Run, Cloud Functions, GKE, Compute Engine, App Engine

[Installation link](https://excalidraw.com/?addLibrary=https%3A%2F%2Fraw.githubusercontent.com%2Fexcalidraw%2Fexcalidraw-libraries%2Fclementbosc-gcp-icons%2Flibraries%2Fclementbosc%2Fgcp-icons.excalidrawlib%3Fraw%3Dtrue)

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -1203,5 +1203,22 @@
     "created": "2022-01-08",
     "updated": "2022-01-08",
     "version": 2
+  },
+  {
+    "name": "GCP Icons",
+    "description": "Here are some of the main GCP products !",
```
Comment (dwelle) [inline libraries.json:1209]:
```suggestion
    "description": "Several common GCP products logos.",
```

**Comment by dwelle** [comment]
Very cool @clementbosc! Tweaked the description a bit. Let's ship! 🚀

**Comment by clementbosc** [comment]
Thanks @dwelle, I am doing some more, I will push them during the week

**Comment by clementbosc** [comment]
Hi @dwelle , what is the procedure to add icons to an existing library ? (this one actually) Thanks

**Comment by dwelle** [comment]
Hey! Right now, you have to make a PR where you replace the old `.excalidrawlib` file with the new one. As a bonus, you can update the `updated` field in `libraries.json`. You can check this PR to see where the files are located. Lemme know if you need help!

**Comment by clementbosc** [comment]
Thanks ! I've just created the PR https://github.com/excalidraw/excalidraw-libraries/pull/318.

**Pull request merged**